default ribbon overlay produces two errors

### Version Number

2.2.2

### What branch are you on?

master

### Describe the Bug

Two errors in Golden Globe ribbons:
```
[ERROR]    | Overlay Error: imdb_award category_filter attribute invalid: best motion picture, animated must be in in [best director - motion picture, best motion picture - animated, ... long list redacted ... ] |
```
```
[ERROR]    | Overlay Error: imdb_award category_filter attribute invalid: best director, motion picture must be in in [best director - motion picture,  ... long list redacted ... ] |
```

### Relevant Collection/Overlay/Playlist Definition

```yml

```

### Logs

_No response_

--- a/defaults/award/golden.yml
+++ b/defaults/award/golden.yml
@@ -28,6 +28,11 @@ collections:
       event_id: ev0000292
       event_year: all
       category_filter:
+        - best animated feature film
+        - best animated film
+        - best foreign film
+        - best foreign language film
+        - best foreign-language foreign film
         - best motion picture - animated
         - best motion picture - comedy
         - best motion picture - comedy or musical
@@ -37,6 +42,7 @@ collections:
         - best motion picture - musical or comedy
         - best motion picture - non-english language
         - best motion picture, musical or comedy
+        - best picture
       winning: true
 
   Golden Globes Best Director Winners:

--- a/defaults/overlays/ribbon.yml
+++ b/defaults/overlays/ribbon.yml
@@ -78,6 +78,11 @@ overlays:
       event_id: ev0000292
       event_year: all
       category_filter:
+        - best animated feature film
+        - best animated film
+        - best foreign film
+        - best foreign language film
+        - best foreign-language foreign film
         - best motion picture - animated
         - best motion picture - comedy
         - best motion picture - comedy or musical
@@ -86,10 +91,8 @@ overlays:
         - best motion picture - musical
         - best motion picture - musical or comedy
         - best motion picture - non-english language
-        - best motion picture, animated
-        - best motion picture, drama
         - best motion picture, musical or comedy
-        - best motion picture, non-english language
+        - best picture
       winning: true
 
   Golden Globe Best Director:
@@ -101,7 +104,6 @@ overlays:
       category_filter:
         - best director
         - best director - motion picture
-        - best director, motion picture
       winning: true
 
   BAFTA Winners:

